README.md

```diff
@@ -63,8 +63,8 @@ s = shell.quote(p)
 * [common_settings](docs/common_settings_doc.md)
 * [directories](docs/copy_directory_doc.md)
     * [directory](docs/directory_doc.md)
-    * [directory_glob](docs/directory_glob.md)
-    * [subdirectory](docs/subdirectory_doc.md)
+    * [directory_glob](docs/directory_glob_doc.md)
+    * [directory_subdirectory](docs/directory_subdirectory_doc.md)
```

Comment:
Don't change the name from `subdirectory` to `directory_subdirectory`. Apparently, `directory_subdirectory_doc.md` is the name of the documentation page but not of the rule itself (see the .bzl file and the title of the md page).

Reply:
Yes, I have already pushed an update commit.

Reply:
Whoops, thanks.